src/app/static/templates/404.ftl

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+    <title> 404 Error Page</title>
+    <link rel="shortcut icon" href="/public/favicon.ico"/>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+    <!-- inject:css -->
+    <link rel="stylesheet" href="/public/css/style-33eddb4b7c3a4150e40d70284fba3cac.min.css">
+    <!-- endinject -->
+</head>
+<body>
+<div>
+<br>
+<br>
+<br>
+ <h1 class="text-center mt-3">404 Error Page </h1>
+ <p class="text-center mt-3">The requested page was not found, try again.</p>
```

Comment:
"Try again" seems a little brusque! Don't think we need it anyway. Would be nice to include a link to the homepage.